### ISSUE TITLE
Web: follow html/javascript redirects

### ISSUE THREAD

**Issue by Mikaela** [ProgVal/Limnoria#1120]
It would be more useful if title could follow through html/javascript redirects instead of saying only the title of redirecting page as the bot should be able to look into those tags like it looks into `<title>`.

* HTML redirect: `<meta http-equiv=refresh content="0; url=https://github.com/Antergos/Cnchi/issues/303#issuecomment-94838586">`
* Javascript redirect:`<script>location='https://github.com/Antergos/Cnchi/issues/303#issuecomment-94838586'</script>`

This is took from jekyll-redirect page https://mikaela.info/r/cnchigit and the whole page from curl seems to be:

```html
 <!DOCTYPE html>
<meta charset=utf-8>
<title>Redirecting...</title>
<link rel=canonical href="https://github.com/Antergos/Cnchi/issues/303#issuecomment-94838586">
<meta http-equiv=refresh content="0; url=https://github.com/Antergos/Cnchi/issues/303#issuecomment-94838586">
<h1>Redirecting...</h1>
<a href="https://github.com/Antergos/Cnchi/issues/303#issuecomment-94838586">Click here if you are not redirected.</a>
<script>location='https://github.com/Antergos/Cnchi/issues/303#issuecomment-94838586'</script>
```

**Comment by GLolol** [comment]
Meta refresh tags aren't super hard to parse, but JavaScript redirects are a lot more complicated. Since JS is a language in itself, the code won't always be parse-able as plain text, and running a JavaScript interpreter for a title snarfer is simply overkill IMO (and might be insecure too).
